src/object_store/src/object/mem.rs

```diff
@@ -112,10 +113,19 @@ impl ObjectStore for InMemObjectStore {
 impl InMemObjectStore {
     pub fn new() -> Self {
         Self {
-            objects: Mutex::new(HashMap::new()),
+            objects: Arc::new(Mutex::new(HashMap::new())),
         }
     }
 
+    /// Create a shared reference to the in-memory object store in this process. Used for multiple
+    /// compute-nodes or compactors in the same process by `risedev playground`.
```

Comment:
I think we should add a feature in the crate to enable this.

Reply:
This function should not be used by default, so `SHARED` should not be initialized and there's no side effect. Adding a feature for this may make playground deployment complicated. 🥵

Reply:
Added explicit notes for this and make it private in the super module.